README.md

```diff
@@ -1,44 +1,47 @@
-# eslint-plugin-console-saas-rules
```

Comment:
Not sure why the diff looks like this. Just added a "DEPRECATED" warning at the top of the file with a link to bitbucket.

Reply:
the "rich diff" looks quite nice

Reply:
Fancy!